csharp/test/Ice/proxy/AllTests.cs

```diff
@@ -1091,32 +1091,6 @@ public static async Task<Test.MyClassPrx> allTests(global::Test.TestHelper helpe
                 // Opaque endpoint encoded with 1.1 encoding.
                 Ice.ObjectPrx p2 = communicator.stringToProxy("test -e 1.1:opaque -e 1.1 -t 1 -v CTEyNy4wLjAuMeouAAAQJwAAAA==");
                 test(communicator.proxyToString(p2) == "test -t -e 1.1:tcp -h 127.0.0.1 -p 12010 -t 10000");
-
-                if(communicator.getProperties().getPropertyAsInt("Ice.IPv6") == 0)
```

Comment:
It's not clear why you removed this test.

Reply:
Restored and updated